projects/detr3d/configs/detr3d_res101_gridmask_dev-1.x.py

```diff
@@ -0,0 +1,241 @@
+_base_ = [
+    # '.../mmdetection3d/configs/_base_/datasets/nus-3d.py',
+    '../../../configs/_base_/default_runtime.py'
```

Comment:
```suggestion
    'mmdet3d::_base_/default_runtime.py'
```
Please modify all the same code.